src/com/untamedears/PrisonPearl/CombatTagListener.java

```diff
@@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
         if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
             return;
         }
-        UUID plruuid = event.getPlayerUUID();
+        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
```

Comment:
That change isn't necessary if it was working the other way, I was getting
errors though and couldn't compile to test

On Friday, February 27, 2015, rourke750 <notifications@github.com> wrote:

> In src/com/untamedears/PrisonPearl/CombatTagListener.java
> <https://github.com/Civcraft/PrisonPearl/pull/34#discussion_r25509686>:
>
> > @@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
> >          if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
> >              return;
> >          }
> > -        UUID plruuid = event.getPlayerUUID();
> > +        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
>
> PP has a soft dependency on namelayer so you need to check if namelayer is
> installed first. Look at some of the other places where nameAPi checks are
> called.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Civcraft/PrisonPearl/pull/34/files#r25509686>.
>